bin/apply-migrations

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Script that invokes flyway migrations.
+# Can be run locally or during a build.
+# In the latter case the jar path *must* be specified.
+# Locally the script will default to building the jar locally and executing it
+# from there.
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+source "$script_dir/../bin/lib/script-helpers"
```

Comment:
can we use some base path parameter initialised and add other relative path from there ?

Reply:
Using this approach means the scripts can find this location irrespective of where they are. The script_dir will always be set to the location of the script and from there we can reliably locate the `script-helpers` find. I've used this approach before and it's usually the easiest way to go.